### DISCUSSION TITLE
Dúvida - nota fiscal de servidor

### DISCUSSION THREAD

**Discussion by alynnefs** [apyb/comunidade#141] [Geral]
A geração da nota fiscal precisa ser na data do evento ou o faturamento pode ser antes? Porque eu queria pedir o pagamento de 50%, mas o servidor não consegue gerar com data no futuro.

**Comment by cecivieira** [comment]
Pode emitir a nota fiscal com data anterior. Basta descerver na nota a data que o serviço será prestado.

**Reply by alynnefs** [reply]
Obrigada <3
